.ci_support/linux_.yaml

```diff
@@ -9,4 +9,4 @@ pin_run_as_build:
     min_pin: x.x
     max_pin: x.x
 python:
-- '2.7'
+- '3.7'
```

Comment:
This will revert to `2.7` again if you rerender before the `conda-forge-pinning` PR got merged/released.

Reply:
This still holds true since we had to revert that `conda-forge-pinning` PR, unfortunately.